Core.Collectors/Config/ConfigManager.cs

```diff
@@ -95,7 +95,7 @@ protected JToken SelectConfigToken(string jsonPath)
             return this.config.SelectToken(jsonPath);
         }
 
-        private void ValidateSettingsExist()
+        protected void ValidateSettingsExist()
```

Comment:
Just checking: change from private to protected is intentional?

Reply:
Yes, this reverts the previous changes with the exception of changing `configManager` and `ValidateSettingsExist()` from private to protected. I needed to do so to set the API domain in the child class constructor and `ValidateSettingsExists()` in `GithubConfigManager.GetApiDomain()`